snap/local/derper-service

```diff
@@ -21,4 +21,5 @@ add_option hostname
 add_option stun-port
 add_flag verify-clients
 
-exec "${SNAP}/bin/derper" "${args[@]}"
```

Comment:
This is not daemon, so what will happened if I run it with `-c <my-path>`? Do we need to handle it?

Reply:
This [is a daemon](https://github.com/canonical/derper-snap/blob/e08d53cf98cd5c6898260750d3a5cac7c3934cfb/snap/snapcraft.yaml#L37), and also this script does not read args from the command line.